### PR TITLE
chore(metadata): add BMF, fix links, fix typos

### PR DESCRIPTION
**Pull request by danielasubotic**

### CODE REVIEW

**Review by mpro7 — APPROVED** [review]
Looks good, thanks!

File: services/metadata/backend/data/db.json [hunk 1]
```diff
@@ -1892,7 +1892,7 @@
                     ],
                     "url": [
                         {
-                            "name": "ark.dasch.sw",
+                            "name": "http://ark.dasch.swiss/ark:/72163/1/0826",
                             "type": "https://schema.org/URL",
                             "url": "ark.dasch.swiss/ark:/72163/1/0826"
```
Comment (mpro7) [inline services/metadata/backend/data/db.json:1897]: protocol is missing here

**Review by mpro7 — CHANGES_REQUESTED** [review]

File: services/metadata/backend/data/db.json [hunk 2]
```diff
@@ -2236,15 +2236,15 @@
...
                     "discipline": [
-                        "http://skos.um.es/unesco6/630110 ",
```
Comment (BalduinLandolt) [inline services/metadata/backend/data/db.json:2239]: this should be a link:
```
{
  "name": "SKOS etc.",
  "type": "...",
  "url": "..."
}
```

**Review by BalduinLandolt — CHANGES_REQUESTED** [review]
Looks mostly good, just what Marcin pointed out, and that one link...

**Review by BalduinLandolt — APPROVED** [review]
thanks!

**Review by mpro7 — APPROVED** [review]

**Pull request merged**